src/API/WesternStatesWater.WestDaat.Accessors/NldiAccessor.cs

```diff
@@ -19,12 +19,18 @@ public NldiAccessor(NldiConfiguration configuration, IUsgsNldiSdk usgsNldiSdk, I
 
         private readonly IUsgsNldiSdk _usgsNldiSdk;
         private readonly NldiConfiguration _configuration;
-        private readonly Dictionary<NavigationMode, string> _featureTypes = new Dictionary<NavigationMode, string>()
+        private readonly Dictionary<NavigationMode, string> _navigationModeName = new Dictionary<NavigationMode, string>()
         {
-            { NavigationMode.UpstreamMain, "UpstreamMain" },
-            { NavigationMode.UpstreamTributaries, "UpstreamTributaries" },
-            { NavigationMode.DownstreamMain, "DownstreamMain" },
-            { NavigationMode.DownstreamDiversions, "DownstreamDiversions" }
+            { NavigationMode.UpstreamMain, "Main.Upstream" },
+            { NavigationMode.UpstreamTributaries, "Tributary.Upstream" },
+            { NavigationMode.DownstreamMain, "Main.Downstream" },
+            { NavigationMode.DownstreamDiversions, "Tributary.Downstream" }
```

Comment:
```suggestion
            { NavigationMode.DownstreamDiversions, "Diversions.Downstream" }
```
??

Just checking that this is Tributary and not Diversion

Reply:
It is normally Diversion going down to the NLDI service.  However, in this case, I want it to match the upstream because we are going to use it for formatting on the frontend.